staging/src/k8s.io/client-go/rest/with_retry.go

```diff
@@ -123,6 +125,8 @@ func (r *withRetry) NextRetry(req *http.Request, resp *http.Response, err error,
 	}
 
 	r.attempts++
+	updateRetryURLMetrics(context.TODO(), req, resp, err)
```

Comment:
I don't think that's the appropriate place for it, this function returns retry parameters, whether a retry should take place. For example, for a request that succeeds the first time, the retry metric would record one retry.

Reply:
yeah, that was my feeling, I was going to ask you about this because I wasn't sure
thanks for confirming

Reply:
removed commit from this PR